diff --git a/lua/obsidian/itertools.lua b/lua/obsidian/itertools.lua
index d146188..588a20d 100644
--- a/lua/obsidian/itertools.lua
+++ b/lua/obsidian/itertools.lua
@@ -26,7 +26,7 @@ M.iter = function(iterable)
       return function()
         return nil
       end
-    elseif vim.tbl_islist(iterable) then
+    elseif vim.islist(iterable) then
       local i = 1
       local n = #iterable
 
diff --git a/lua/obsidian/util.lua b/lua/obsidian/util.lua
index 6533116..a2e0083 100644
--- a/lua/obsidian/util.lua
+++ b/lua/obsidian/util.lua
@@ -44,7 +44,7 @@ util.tbl_is_array = function(t)
     return false
   end
 
-  return vim.tbl_islist(t)
+  return vim.islist(t)
 end
 
 ---Check if an object is an non-array table.
